test/integration/multisig.js

```diff
@@ -60,16 +63,19 @@ describe('bitcoinjs-lib (multisig)', function () {
       blockchain.t.transactions.propagate(tx.toHex(), function (err) {
         if (err) return done(err)
 
-        // check that the above transaction included the intended address
-        blockchain.t.addresses.unspents(targetAddress, function (err, unspents) {
-          if (err) return done(err)
+        // allow for TX to be processed
+        setTimeout(function () {
```

Comment:
hmm, it's been working fine without it (issues seem unrelated) but wouldn't hurt to have a short timeout.

propagate (and faucet) broadcast TX through bitcoind which then relays it to our custom node, so there is a short delay in getting the TX processed (which is slightly more during stresstests or during block processing).  
normally the overhead of the http requests covers the time it takes to process it, but a short timeout helps, I'll lower it to 200ms.